gantry/simulator/interfaces.cpp

```diff
@@ -60,16 +60,18 @@ static tmc2130::configs::TMC2130DriverConfig driver_configs{
         .GPIO_handle = 0,
     }};
 
+static auto linear_motion_sys_config =
+    lms::LinearMotionSystemConfig<lms::BeltConfig>{
+        .mech_config = lms::BeltConfig{.pulley_diameter = 12.7},
+        .steps_per_rev = 200,
+        .microstep = 32,
```

Comment:
Is 32 the microstepping we're currently using in the gantry project?

Reply:
Seems to be based on https://github.com/Opentrons/ot3-firmware/blob/main/gantry/core/utils.cpp#L20 

HOWEVER, that brings up the point that this should just get pulled from the core/utils file instead of duplicating the config